sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>This is the Microsoft Azure Application Configuration Service client library</Description>
     <AssemblyTitle>Microsoft Azure.Data.AppConfiguration client library</AssemblyTitle>
-    <Version>1.0.0</Version>
+    <Version>1.0.0-preview</Version>
```

Comment:
The file belongs to AppConfiguration. I don't suppose this change should be checked in.

Reply:
It's `1.0.0` on master.